lib/src/room.dart

```diff
@@ -606,6 +607,12 @@ class KurentoRoom extends ChangeNotifier implements Room {
   }
 
   Future<void> _reconnect() async {
+    if (_isReconnecting) {
+      _log.warning('a reconnect is already in progress');
+      return;
+    }
+    _isReconnecting = true;
+
```

Comment:
Does Dashboard have to do anything in case of reconnections? I assume not, but if it does, this could cause an out of synch state. I'll double check... unless you already know! It would be nice to prevent this on the Dashboard side too.

Reply:
@ipainchaud Dash just sends a `RECONNECT` event, similar to what it does when taking a photo. It doesn't wait for a response. 

Dash already has a reconnect throttle built in, but in this particular case, it looks like it either didn't work or a message got delivered twice